About/About/what-are-chaoss-goals.md

```diff
@@ -1,9 +1,11 @@
 ## What are CHAOSS Goals?
 
-The project aims to:
+The project goals are to:
 
-- Establish standard implementation-agnostic metrics for measuring community activity, contributions, and health
+- Establish standard implementation-agnostic metrics for measuring community health
 
 - Produce integrated open source software for analyzing software community development
 
-- Build reproducible project health reports/containers
+- Develop program for the deployment of metrics not attainable through online trace data
```

Comment:
"Develop program" seems to be missing something

how about: 
```suggestion
- Develop a program for the deployment of metrics not attainable through online trace data
```

Reply:
how about programs? I think I just forget an s

Reply:
Yes, that works too :)